apps/contribution/test/contribution.js

```diff
@@ -45,6 +45,16 @@ const mineBlock = function() {
   });
 };
 
+const getBlockNumber = function() {
```

Comment:
getBlockNumber should actually return a promise as well, so this entire function is superfluous. We're using it with a promise e.g. here: https://github.com/67P/kredits-web/blob/master/app/services/kredits.js#L150

Reply:
it return also there a promise, no ?

Reply:
The function you added is just wrapping the normal callback in a promise. Wherever it's called you can just use the original function with a promise.

Reply:
I updated it, the thing is blocks get mined instantly so veto period end...